.github/workflows/lint-format.yml

```diff
@@ -66,11 +66,11 @@ jobs:
       - name: Install jinja
         run: python -m pip install jinja2
       - name: Create compile_commands.json
-        run: mkdir build-cmake && cd build-cmake && cmake -DWITH_OLD_COMMANDS=ON -DWITH_EXAMPLES=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=YES ..
+        run: ./gradlew generateCompileCommands -Ptoolchain-optional-roboRio
```

Comment:
Changing to Gradle for this means the jinja install can be removed (it's only needed for cmake).